src/topology/instances/complex.lean

```diff
@@ -75,11 +75,11 @@ metric.uniform_continuous_iff.2 $ λ ε ε0, begin
 end
 
 lemma uniform_continuous_mul (s : set (ℂ × ℂ))
-  {r₁ r₂ : ℝ} (r₁0 : 0 < r₁) (r₂0 : 0 < r₂)
+  {r₁ r₂ : ℝ} (r₁0 : 0 < r₁)
```

Comment:
The assumption `0 < r₁` is also not necessary: unless `s` is empty (in which case the conclusion is obvious), then `0 < r₁` holds. This comes from `rat_mul_continuous_lemma`, where this assumption can also be removed.